jscomp/super_errors/super_location.ml

```diff
@@ -0,0 +1,34 @@
+open Misc
+open Asttypes
+open Parsetree
+open Types
+open Typedtree
+open Btype
+open Ctype
+
+open Format
+open Printtyp
+
+(* taken from https://github.com/ocaml/ocaml/blob/4.02/parsing/location.ml#L337 *)
+(* we override the default reporter with this one *)
+let rec better_error_reporter ppf ({Location.loc; msg; sub; if_highlight} as err) =
```

Comment:
can you add a line of comment about what changes you did compared with Location.default_error_reporter, since they are mostly the same it is really hard to tell